spec/factories/user_to_answer_ratings.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user_to_answer_rating do
+    
```

Comment:
where are attributes here? if you don't use this factory, just remove it.

Reply:
Completed.